manifests/01-registry-credentials-request-gcs.yaml

```diff
@@ -11,4 +11,7 @@ spec:
     namespace: openshift-image-registry
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
-    kind: GCSProviderSpec
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/storage.admin
+    - roles/iam.serviceAccountUser
```

Comment:
As https://github.com/openshift/cloud-credential-operator/pull/93 merged
@coreydaley can you add `skipServiceCheck: true` ...

Reply:
Added in current push, ptal